env.txt

```diff
@@ -0,0 +1,12 @@
+DATABASE_NAME=prosopo
+DATABASE_IMAGE=mongo:5.0.4
+DATABASE_PORT=27017
+DATABASE_HOST=localhost
+MONGO_INITDB_ROOT_USERNAME=root
+MONGO_INITDB_ROOT_PASSWORD=root
+MONGO_INITDB_DATABASE=prosopo
+PROVIDER_MNEMONIC=napkin deal since limit speed fit unveil uniform crazy tape cable will
+CONTRACT_ADDRESS=5GAToDy7LM9NWhudRxa2aZBoMNPyhzqVnu1cqEEFaQH45nU9
+DAPP_CONTRACT_ADDRESS=5DQPHYGZCVNKJCV9kMrV9tHaR5uh3qLutkDqyztJHxkXNjmK
+API_BASE_URL=http://localhost:3000
+SUBSTRATE_NODE_URL=ws://localhost:9944
```

Comment:
Potentially conflicting with https://github.com/prosopo-io/integration/blob/bb4193e9528b192c3147bd12e53bb0a9d34f72c8/env.txt#L14-L15 but we could have both